bilge-impl/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(iter_intersperse)]
```

Comment:
uhhh, this is nightly-only
I remember I used `fold` or `reduce` instead I think.

Reply:
yeah, not sure what I was thinking there, will change